examples/simple_fleet_app_operator_permissions/main.tf

```
@@ -1,5 +1,5 @@
 /**
- * Copyright 2024 Google LLC
+ * Copyright 2025 Google LLC
```

The original date should be preserved ([go/copyright#copyright-notice](http://go/copyright#copyright-notice)), if you wish to update this can be used:

```suggestion
 * Copyright 2024-2025 Google LLC
```